### PR TITLE
UI design merge into Development

### PR DESCRIPTION
**Pull request by Snotvlek**
everything is now working, except the search bar, current weather info and the weather alert section

### CODE REVIEW

File: TWeather.py [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+   # nicify text (NOT USED RIGHT NOW)
+   #text = textwrap.fill(tweet['text'],initial_indent='    ', subsequent_indent='    ')
+   
+   #update the weather graph based on location data.
+   
+   # generate output
+   #print(tweet['user']['name'])
+   emit('regular', tweet)
+
+@event('weather_update_rawtext_update')
```
Comment (bogdanmois23) [inline TWeather.py:59]: Why is this?

**Review by NiekAukes — CHANGES_REQUESTED** [review]
just a few things, get rid of Tweather_static_old can be deleted
and maybe comment on some new things

File: TWeather.py [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+from eca.generators import start_offline_tweets
+import datetime
+import textwrap
+from WeatherEventGen import *
+import Classifier
+import WeatherCondition
+
+#Adding a handler for the search button press.
+def add_request_handlers(httpd):
```
[the remark this answers is not in the document]
Comment (Snotvlek) [reply TWeather.py:12]: for making the searchbutton work later on

File: TWeather.py (hunk 1 above)
Comment (Snotvlek) [reply TWeather.py:59]: template function for me to test, can be removed

**Review by Snotvlek — COMMENTED** [review]
looks good

**Review by Snotvlek — COMMENTED** [review]
Yep seems legit

**Comment by Snotvlek** [comment]
> 
> 
> just a few things, get rid of Tweather_static_old can be deleted and maybe comment on some new things

Yes, please, and then it can merge. But I cannot figure out how to merge right now so you do it @NiekAukes

**Review by NiekAukes — APPROVED** [review]

**Review by bogdanmois23 — APPROVED** [review]

**Pull request merged**